### DISCUSSION TITLE
Development environment, Code changed, page not refreshed

### DISCUSSION THREAD

**Discussion by piewaer** [apache/superset#15756] [Q&A / Help]
React hot load has no effect, the code is changed, and the page is not updated synchronously

**Comment by nytai** [comment]
Can you provide more info on how you’re running superset?

**Reply by piewaer** [reply]
in frontend,npm run dev-server, JSX cannot parse
![image](https://user-images.githubusercontent.com/87263497/126145801-46d73fbb-c88d-4a50-af81-af17d58a4ed9.png)


superset-ui can't be loaded correctly. It should be the version of superui, but the installation uses the default version
![image](https://user-images.githubusercontent.com/87263497/126146223-51230953-fdb6-4411-9fbd-2fcba9ecf68f.png)
